.circleci/config.yml

```
@@ -106,6 +194,18 @@ workflows:
           context: hmpps-common-vars
           requires:
             - build
+      - integration_test:
+          context: hmpps-common-vars
+          requires:
+            - build
+      - accessibility_test:
+          context: hmpps-common-vars
+          requires:
+            - build
+      - e2e_test:
+          context: hmpps-common-vars
+          requires:
+            - build
```

In the event that we end up running all these tests in the pipeline, it's probably worth making this one depend on the accessibility-tests (or vice versa). This should avoid possible port collisions when running npm start and would mean we don't need to wait for the database to load twice

```suggestion
      - accessibility_test
```